apps/heft/includes/jest-shared.config.json

```diff
@@ -3,9 +3,11 @@
 
   "rootDir": "../../../../",
   "testURL": "http://localhost/",
-  "testMatch": ["<rootDir>/src/**/*.test.ts?(x)"],
+  "testMatch": ["<rootDir>/src/**/*.test.{ts,tsx}"],
+  "transformIgnorePatterns": [],
   "transform": {
-    "src[\\\\/].+\\.ts$": "<rootDir>/node_modules/@rushstack/heft/lib/plugins/JestPlugin/HeftJestSrcTransform.js"
```

Comment:
Note: This style of RegExp was not very readable.  Also, because the filtering happens after `testMatch` (plus any recursively require()`'d paths), it is not necessary to include `src` in the pattern.